src/main/java/se/liu/ida/hefquin/engine/federation/access/impl/reqproc/GraphQLRequestProcessorImpl.java

```diff
@@ -26,14 +41,61 @@ public GraphQLRequestProcessorImpl() {
 	public JSONResponse performRequest( final GraphQLRequest req,
 	                                    final GraphQLEndpoint fm )
 	                                    		throws FederationAccessException {
-		// TODO this function needs to be implemented
-		//
-		// To get the GraphQL query that is to be executed you can do
-		// req.getGraphQLQuery() and to get the URL of the GraphQL
-		// endpoint to which the request needs to be posted you can
-		// do fm.getInterface().getURL()
-
-		return null;
-	}
 
+		Date startTime = new Date();
```

Comment:
For all variables that are not re-assigned later, please make them `final`, like this:
```suggestion
		final Date startTime = new Date();
```
Notice that this suggestion can be directly committed from this web interface, and then you can `git pull` to get the changes in your local repository.